deploy/02-deploy-comptroller-set-extensions.ts

```diff
@@ -100,7 +114,7 @@ const func: DeployFunction = async ({ viem, getNamedAccounts, deployments }) =>
     } else {
```

Comment:
It may be beneficial to add `compPrudentiaExtension.address` in the args above on line 111.